synapse/config/sso.py

```diff
@@ -67,6 +67,11 @@ def read_config(self, config, **kwargs):
         login_fallback_url = self.public_baseurl + "_matrix/static/client/login"
         self.sso_client_whitelist.append(login_fallback_url)
 
+        # experimental support for MSC2858 (multiple SSO identity providers)
+        self.experimental_msc2858_support_enabled = config.get(
+            "experimental_msc2858_support_enabled", False
```

Comment:
I don't know if we've reached any conclusions on what name this sort of setting should have. Hopefully it's temporary so doesn't matter.

Reply:
:+1: let's go with `experimental_features`.